edx_rbac/utils.py

```diff
@@ -151,3 +179,59 @@ def append_role_auth_claim(role_string, context=None):
             else:
                 append_role_auth_claim(role_string)
     return role_auth_claim
+
+
+def is_iterable(obj):
+    """
+    Returns True if obj has an __iter__() method, False otherwise.
+    """
+    try:
+        iter(obj)
+        return True
+    except TypeError:
+        return False
```

Comment:
Minor thing that maybe we don't care about at all, but I think I've run into it before: from https://stackoverflow.com/questions/1952464/in-python-how-do-i-determine-if-an-object-is-iterable, this might say that some test objects are iterable when they actually aren't and it recommends using
```python
from collections.abc import Iterable   # drop `.abc` with Python 2.7 or lower

def is_iterable(obj):
    return isinstance(obj, Iterable)
```
just a small thought

Reply:
oooh, TIL, thanks!